app/helpers/application_helper.rb

```diff
@@ -206,7 +206,7 @@ def searchable?
 
   def auto_complete_search(name, val, options = {})
     path = options[:path] || send("#{controller_name}_path")
```

Comment:
Perhaps I am missing something but it looks like [this](https://github.com/theforeman/foreman/tree/develop/app/views/filters/_form.html.erb#L51-51) is the only place where `path` is used and it's already a full path.  So I don't understand adding the complexity of another parameter.  Either you specify a full path or it defaults.